app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,70 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+
```

Comment:
- [ ] Test specs are missing. Kindly ensure to test your application using _RSpec_ and _Capybara_ testing libraries.